geode-docs/reference/topics/client-cache.html.md.erb

```diff
@@ -338,6 +338,32 @@ Provide a server list or `locator` list, but not both.
        port="123456"/>
 </pool>
 ```
+## <a id="cc-socket-factory" class="no-quick-link"></a>&lt;socket-factory&gt;
+
+Defines a factory to create socket connections to locators and servers.  A typical use of this element is to redirect connections to an ingress gateway such as Istio or HAProxy in a cluster where the TLS (SSL) Server Name Indication (SNI)field is set to indicate the actual locator or server the client is trying to reach.  This allows you to expose only the gateway hostname:port without the client needing to be able to resolve the names of the locator and server machines.
```

Comment:
formatting: I believe you need a space in "(SNI)field" -> "(SNI) field"

Reply:
done